server-landscape-corrected.yaml

```diff
@@ -0,0 +1,145 @@
+heat_template_version: 2013-05-23
```

Comment:
I tested the newest heat template version of 2016-10-14 which gave me this Error:
`ERROR: The template version is invalid: "heat_template_version: 2016-10-14". "heat_template_version" should be one of: 2013-05-23, 2014-10-16, 2015-04-30, 2015-10-15, 2016-04-08`
I tested the suggested `2016-04-08` template and it works fine.

Do you need the specific `2015-10-15` version? If not, you could use the newer version too.